service/openapi.yaml

```diff
@@ -0,0 +1,241 @@
+openapi: 3.1.0
+info:
+  version: 0.1.0
+  title: OpenFeature Remote Evaluation Protocol (OFREP)
+  description: OFREP define the protocol for remote flag evaluations
+  contact:
+    url: https://github.com/open-feature/protocol
+  license:
+    identifier: Apache-2.0
+    name: Apache 2.0
+security:
+  - ApiKeyAuth: [ ]
+  - BearerAuth: [ ]
+paths:
+  /ofrep/v1/evaluate/flags/{key}:
+    post:
+      description: OFREP single flag evaluation request
+      parameters:
+        - name: key
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/evaluationRequest'
+      responses:
+        '200':
+          description: OFREP successful evaluation response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/evaluationSuccess'
+        '400':
+          description: Bad evaluation request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/evaluationFailure'
+        '401':
+          description: Unauthorized - You need credentials to access the API
+        '403':
+          description: Forbidden - You are not authorized to access the API
+        '429':
+          description: Rate limit reached on the Flag Management System
+          headers:
+            Retry-Later:
+              description: Indicates when to retry the request again
+              schema:
+                type: string
+                format: date-time
+                examples:
+                  - '2024-02-07T12:00:00Z'
+        '500':
```

Comment:
These errors have no schema and the default is just `"string"`. Should we include `#/components/schemas/evaluationFailure` in these responses? Same question for the other endpoint.

Reply:
If we see the need for an error payload, for example for logging purposes, we could include one with at least an error message

Reply:
If no objections, I will define this and add to be used by appropriate 4xxx & 5xx responses

Reply:
This is complete with the commit 10eda958c989cfc9ad79b89d9b302ade684f0f5b